client/src/components/MediaCard/MediaCard.js

```diff
@@ -0,0 +1,26 @@
+
+import React from 'react';
+import { Link } from "react-router-dom";
+import "./MediaCard.css";
+
+// Function based React Component
+const MediaCard = (props) => {
+
+  // Default Class to apply to Component
+
+  let styles = {
+    backgroundImage: `url(${props.imageSrc})`,
+    backgroundSize: 'cover',
+  }
+
+  return (
+  
+    <Link className={`MediaCard MediaCard--${props.size}`} to={props.to} style={styles}>
```

Comment:
Look up default props in React so that you can have a default size for your component.